docs/articles/documentation/test-api/actions/action-options.md

```diff
@@ -10,6 +10,7 @@ This topic describes [test action](README.md) options.
 
 * [Basic Action Options](#basic-action-options)
 * [Mouse Action Options](#mouse-action-options)
+* [DragToElement Options](#dragtoelement-options)
```

Comment:
Should link be `drag-to-element`?

Reply:
nope, it's autogenerated links. Spaces are replaces with `-`, but there are no spaces in `DragToElement`

Reply:
I have no control over the anchor IDs. As long as we name the section `DragToElement Action Options`, the ID is `dragtoelement-action-options`.

Reply:
Sorry, you right